x/hard/client/rest/tx.go

```diff
@@ -97,7 +97,7 @@ func postRepayHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
-		msg := types.NewMsgRepay(req.From, req.Amount)
+		msg := types.NewMsgRepay(req.From, req.Owner, req.Amount)
```

Comment:
similar to CLI suggestion, could parse the request and if `Owner` is empty, use `From`.

Reply:
Updated REST in https://github.com/Kava-Labs/kava/pull/801/commits/4d595c50d47153e9a2a2a4be72475a32f63884d1